python/cudf/cudf/tests/test_concat.py

```diff
@@ -661,13 +658,92 @@ def test_concat_join(objs, ignore_index, sort, join, axis):
     )
 
 
+@pytest.mark.parametrize(
+    "objs",
+    [
+        [
+            pd.DataFrame(
+                {
+                    "x": range(10),
+                    "y": list(map(float, range(10))),
+                    "z": [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                }
+            ),
+            pd.DataFrame(
+                {"x": range(10, 20), "y": list(map(float, range(10, 20)))}
+            ),
+        ],
+    ],
+)
 @pytest.mark.parametrize("ignore_index", [True, False])
 @pytest.mark.parametrize("sort", [True, False])
 @pytest.mark.parametrize("join", ["inner", "outer"])
-@pytest.mark.parametrize("axis", [0, 1])
-@pytest.mark.xfail(
-    raises=NotImplementedError, reason="we do not support duplicate columns"
+@pytest.mark.parametrize("axis", [1])
+def test_concat_join_axis_1_dup_error(objs, ignore_index, sort, join, axis):
+    gpu_objs = [gd.from_pandas(o) for o in objs]
+    # we do not support duplicate columns
+    with pytest.raises(NotImplementedError):
+        assert_eq(
+            pd.concat(
+                objs,
+                sort=sort,
+                join=join,
+                ignore_index=ignore_index,
+                axis=axis,
+            ),
+            gd.concat(
+                gpu_objs,
+                sort=sort,
+                join=join,
+                ignore_index=ignore_index,
+                axis=axis,
+            ),
+        )
```

Comment:
With testing for this error do we need to parametrize all of the combinations?

Reply:
I don't think we do! I'll remove some of the extra parameters :D